include/z64.h

```diff
@@ -670,9 +670,38 @@ typedef struct {
     /* 0x144 */ Vec3f rot;
 } SkyboxContext; // size = 0x150
 
+typedef enum {
+    MESSAGE_ICON_TRIANGLE,
+    MESSAGE_ICON_SQUARE,
+    MESSAGE_ICON_ARROW
+} MessageBoxIcon;
+
+typedef struct {
+    /* 0x0000 */ u32  offset;
+    /* 0x0004 */ u32  size;
+    /* 0x0008 */ u8   xy;
+} MessageData;
```

Comment:
What's the reason behind this struct? I think we should avoid sub structs unless they serve a real purpose.
Also what's `xy`? I don't see any usage of it here. And actually a lot of things in the font struct seem unused. Same goes for the box icon enum.

Reply:
The `MessageData` struct is from `z_message_PAL` progress, of which I copied part of my `MessageContext` struct from (everything before `textboxSegment`) when starting the file. It's possibly removable now given that it's also the start of the `Font` struct. These fields are confirmed but I can remove them if they are unwanted at this time. The box icon enum comes from inspecting the textures in `message_static`, they are supposed to be used in the `icon` argument of `LoadMessageBoxEndIcon`.

Reply:
I see. In that case, I think it would be better to leave only the stuff necessary for z_kanfont.c and such, if possible. This way we can review the other stuff once we can see where it's used in a future message PR.